tests/test_syntax_sugar.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from gem import gem
+
+
+def test_expressions():
+    x = gem.Variable("x", (3, 4))
+    y = gem.Variable("y", (4, ))
+    i = gem.Index()
+    j = gem.Index()
```

Comment:
One more UFL imitation that might make sense to adopt is `indices`, so the above two lines could be replaced with:

    i, j = gem.indices(2)